development/progress-bar-app/pages/progress-bar/index.mdx

```diff
@@ -0,0 +1,91 @@
+---
+HEAD OBJECT
+title: "ReuseJS progress-bar"
+author: "Md Amir Sohail"
+---
+
+import RoundedProgress  from "../../components/progress-bar/RoundedProgress/RoundedProgress.tsx"
+import UploadDownloadProgress  from "../../components/progress-bar/UploadDownloadProgress/UploadDownloadProgress.tsx"
+import Progress  from "../../components/progress-bar/Progress/Progress.tsx"
+import CircularProgress  from "../../components/progress-bar/CircularProgress/CircularProgress.tsx"
+
+## Table of contents
+
+## Introduction
+
+*`ProgressBar` component enhance websites by providing visual feedback for file uploads, form submissions, content loading, task completion, downloads/updates, and page scrolling. They improve the user experience, engagement, and interface, keeping users informed and motivated in a concise and intuitive manner.
```

Comment:
How can this progress bar be used for page scrolling? Can you please add an example for this. It will be a useful use case.

Reply:
Should i do this, it will take time to implement this with pageScroll progress. also we need to create a separate issue for this, after merging running PR's.

Reply:
Yes please implement this too. As this is a very popular use case and if be can implement it using this component and provide an example for this it will be very useful.

Reply:
Example added please review